dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -10246,63 +10263,92 @@ components:
       - isValid
       - validatorTool
       - validatorToolVersion
-    ValidationInfo:
+    ValidationStatusMetric:
       type: object
-      description: Aggregated validation information
+      description: Aggregated metrics about workflow validation statuses
       properties:
-        failedValidationVersions:
-          type: array
-          description: A set of validator tool versions that unsuccessfully validated
-            the workflow in its most recent run
-          items:
-            type: string
-            description: A set of validator tool versions that unsuccessfully validated
-              the workflow in its most recent run
-        mostRecentErrorMessage:
-          type: string
-          description: The error message of the most recent validation run if it failed
-        mostRecentIsValid:
-          type: boolean
-          description: The boolean isValid value from the most recent validation run
-        mostRecentVersion:
+        id:
+          type: integer
+          format: int64
+          description: Implementation specific ID for the count metrics in this webservice
+        validatorTools:
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/ValidatorInfo'
+          description: A map containing key-value pairs indicating whether the validator
+            tool successfully validated the workflow
+      required:
+      - validatorTools
+    ValidatorInfo:
```

Comment:
Could use examples or a better description  (same idea as the javadoc above, this is storing for a particular validator like miniwdl whereas ValidatorVersionInfo is storing for a particular version of it (miniwdl 1.0)

Reply:
The word `version` in particular is used a lot in different contexts, so this should emphasize/document that this is the validator version and not a workflow version